src/GovBR/Provider.php

```diff
@@ -131,12 +131,6 @@ public static function additionalConfigKeys()
      */
     protected function getBaseUrlForEnvironment(): string
     {
-        $environment = $this->getConfig('environment', 'production');
-
-        return match ($environment) {
-            'staging'    => $this->stagingUrl,
-            'production' => $this->productionUrl,
-            default      => throw new RuntimeException("Invalid environment '{$environment}' selected for GovBR provider."),
-        };
+        return config('app.env') === 'production' ? $this->productionUrl : $this->stagingUrl;
```

Comment:
This would be a breaking change right? This might cause apps to not function correctly, so I don't think this is a good idea